### PR TITLE
Replace current Sidebar with Ant Design Sidebar

### PR DESCRIPTION
**Pull request by EDsCODE**
Starting to use ant design for components

- adds antd package
- replaces the sidebar with an antd based sidebar

### CODE REVIEW

**Comment by EDsCODE** [comment]
Could use some initial QA on deployment just to see what it looks like

File: frontend/src/lib/components/InviteTeam.js [hunk 1]
```diff
@@ -26,9 +23,8 @@ export class InviteTeam extends Component {
                         type="text"
                         ref={this.urlRef}
                         className="form-control"
-                        value={
-                            url + '/signup/' + this.props.user.team.signup_token
-                        }
+                        value={url + '/signup/' + this.props.user.team.signup_token}
+                        readOnly={true}
```
Comment (EDsCODE) [inline frontend/src/lib/components/InviteTeam.js:27]: fixes a lingering input error related to this component

**Comment by timgl** [comment]
This looks great and now works. However, I'm a little worried about the massive increase in bundle size. 

With antd:
image

On master:
image

200kb more for the JS bundle is a little worrying, and I think we can implement [tree shaking](https://webpack.js.org/guides/tree-shaking/) to solve that. 

The additional 600kb of CSS is super noticable, when you refresh now the page shows some unstyled buttons (so the React has rendered) before the css is even finished loading. I think we should move the css import to `styles.scss` and do the same thing we did with bootstrap: load css only for the components we actually need, one at a time. It's a little bit more effort but will really speed up initial load and subsequent performance (b/c browser has to parse less css).

EDIT: happy for the action out of this comment to be: add another checkbox to the main issue and merge this in the mean time

**Comment by EDsCODE** [comment]
good call. I'll add the todo and look at a few ways to do reduce the css. ant-design offers a few modularizing alternatives that'll be allow for even less overhead

**Review by timgl — APPROVED** [review]

**Pull request merged**